virttest/utils_v2v.py

```diff
@@ -606,8 +606,23 @@ def get_driver_info(self, signed=True):
         cmd = "DRIVERQUERY"
         if signed:
             cmd += " /SI"
-        status, output = self.session.cmd_status_output(cmd)
-        return self.run_cmd(cmd)[1]
+        # Try 5 times to get driver info
+        output, count = '', 5
+        while count > 0:
+            logging.debug('%d times remaining for getting driver info' % count)
+            try:
+                output = self.session.cmd_output(cmd)
+            except Exception as detail:
+                logging.error(detail)
+                count -= 1
+                # Clean up output
+                self.session.cmd('cls')
+            else:
+                break
+        if not output:
+            logging.error('Fail to get driver info')
+        logging.debug("Command output:\n%s", output)
+        return output
 
```

Comment:
It's not really that utils_misc.wait_for **only** deals with functions which need more time to get result.
We can make use of wait_for to check the cases that doesn't need consume time by decreasing the parameter ‘step‘.
And you need less than 3 times to get right result. maybe other guys needs more times, or just only once.
Anyhow, you try a fixed times can fix your problem.  That's just my advice.